modules/installation-configuration-parameters.adoc

```diff
@@ -767,10 +767,16 @@ host must trust the certificate.
 |A map of keys and values that the installation program adds as tags to all resources that it creates.
 |Any valid YAML map, such as key value pairs in the `<key>: <value>` format. For more information about AWS tags, see link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/Using_Tags.html[Tagging Your Amazon EC2 Resources] in the AWS documentation.
 
+[NOTE]
+====
+You can add upto 25 user defined tags during installation. The remaining 25 tags are reserved for {product-title}.
```

Comment:
- Replace "upto" with "up to."
- The third cell for this note expands the column off screen.
<img width="783" alt="image" src="https://user-images.githubusercontent.com/83772007/193809378-35277a32-8733-420c-be5e-dc83951483ba.png">

Reply:
there is a scrolling bar at the bottom of the table